foxtrot-core/src/test/java/com/flipkart/foxtrot/core/querystore/actions/FilterActionTest.java

```diff
@@ -597,6 +598,7 @@ public void testQueryMultipleFiltersOrCombiner() throws FoxtrotException, JsonPr
     }
 
     @Test
```

Comment:
Pagination is not supported anymore
https://www.elastic.co/guide/en/elasticsearch/reference/6.8/breaking-changes-6.0.html#_scroll